source/includes/_payment.md

```diff
@@ -3945,12 +2646,8 @@ account.audit_logs_with_history(options)
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-<
-[
-  {
+```json
+{
     "changeType": "INSERT",
```

Comment:
Is the response correct, the response should be an array, I think you missed adding `[`

Reply:
Hi Reshma, fixed all the places where array was required.